test/aria/core/useragent/BrowserAndDeviceTestCase.js

```diff
@@ -346,30 +347,30 @@ Aria.classDefinition({
             forEach(specifications, function(specification) {
                 // -------------------------------------------------- flags sets
 
-                var trueFlag;
+                var trueFlags = [];
                 var falseFlags = [].concat(specification.flags);
 
                 var expectedFlagName = expectedValues[specification.category];
                 if (expectedFlagName != null) {
-                    trueFlag = expectedFlagName;
-                    falseFlags.splice(aria.utils.Array.indexOf(falseFlags, trueFlag), 1);
+                    trueFlags = isArray(expectedFlagName) ? expectedFlagName : [expectedFlagName];
+                    forEach(trueFlags, function(trueFlag) {
+                        falseFlags.splice(aria.utils.Array.indexOf(falseFlags, trueFlag), 1);
+                    });
                 }
 
                 // ------------------------------------------------------- tests
 
-                if (trueFlag != null) {
-                    var propertyName = Browser._buildFlagName(trueFlag);
-                    var message = 'Flag "' + propertyName + '" should be true. ' +
-                    'Browser: ' + useCase.id + ', ua: ' + useCase.ua;
-                    this.assertTrue(actualValues[propertyName], message);
-                }
+                var checkFunction = function(expectedValue) {
+                    return function(falseFlag) {
+                        var propertyName = Browser._buildFlagName(falseFlag);
```

Comment:
You forgot to rename `falseFlag` to `flag` (since there is no more information of expected value in the name).